lib/runtime-c-api/tests/test-import-function.c

```diff
@@ -14,8 +14,15 @@ typedef struct {
   int value;
 } context_data;
 
+struct print_str_context {
+    int call_count;
+};
+
 void print_str(wasmer_instance_context_t *ctx, int32_t ptr, int32_t len)
```

Comment:
This PR is an excellent start, thank you, but it doesn't solve the issue we have with dynamic languages like PHP, Python and so on.

In C, Go, Rust… `print_str` is declared with static arguments (here `ptr` and `len`). In dynamic languages, the function isn't statically declared. Generally, a “function” is a combination of one or more structures (in PHP for instance, this is two, in Python, this is one) and the VM has some mechanisms to call the function. We __cannot__ declare a list of arguments. Thus, an API like https://github.com/WebAssembly/wasm-c-api/blob/040b983b517ff8ef431f9396e8b98b096fa37d28/example/hello.c#L11-L13 where arguments are stored in an array (so are dynamic, opposed to statically declared) is ideal and the only way to make it work.

I don't know if I'm clear. Am I?

Reply:
Fixed in latest commit.